datasets/bbc_hindi_nli/README.md

```diff
@@ -0,0 +1,190 @@
+- Multimodal Digital Media Analysis Lab - Indraprastha Institute of Information Technology, New Delhi, India disclaims any responsibility for the use of the corpus and does not provide technical support. However, the contact listed above will be happy to respond to queries and clarifications.
+- Rather than redistributing the corpus, please direct interested parties to this page
+- Please feel free to send us an email:
+  - with feedback regarding the corpus.
+  - with information on how you have used the corpus.
+  - if interested in having us analyze your data for natural language inference.
+  - if interested in a collaborative research project.
+
+
+### Licensing Information
+
+Copyright (C) 2019 Multimodal Digital Media Analysis Lab - Indraprastha Institute of Information Technology, New Delhi (MIDAS, IIIT-Delhi).
+Pls contact authors for any information on the dataset.
+
+### Citation Information
+
+```
+    @inproceedings{uppal-etal-2020-two,
+    title = "Two-Step Classification using Recasted Data for Low Resource Settings",
+    author = "Uppal, Shagun  and
+      Gupta, Vivek  and
+      Swaminathan, Avinash  and
+      Zhang, Haimin  and
+      Mahata, Debanjan  and
+      Gosangi, Rakesh  and
+      Shah, Rajiv Ratn  and
+      Stent, Amanda",
+    booktitle = "Proceedings of the 1st Conference of the Asia-Pacific Chapter of the Association for Computational Linguistics and the 10th International Joint Conference on Natural Language Processing",
+    month = dec,
+    year = "2020",
+    address = "Suzhou, China",
+    publisher = "Association for Computational Linguistics",
+    url = "https://www.aclweb.org/anthology/2020.aacl-main.71",
+    pages = "706--719",
+    abstract = "An NLP model{'}s ability to reason should be independent of language. Previous works utilize Natural Language Inference (NLI) to understand the reasoning ability of models, mostly focusing on high resource languages like English. To address scarcity of data in low-resource languages such as Hindi, we use data recasting to create NLI datasets for four existing text classification datasets. Through experiments, we show that our recasted dataset is devoid of statistical irregularities and spurious patterns. We further study the consistency in predictions of the textual entailment models and propose a consistency regulariser to remove pairwise-inconsistencies in predictions. We propose a novel two-step classification method which uses textual-entailment predictions for classification task. We further improve the performance by using a joint-objective for classification and textual entailment. We therefore highlight the benefits of data recasting and improvements on classification performance using our approach with supporting experimental results.",
+}
+```
```

Comment:
````suggestion
```

### Contributions

Thanks to [@avinsit123](https://github.com/avinsit123) for adding this dataset.
````
We added a new section :)